src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -42,16 +42,22 @@ internal abstract class AbstractAddParameterCheckCodeRefactoringProvider<
     where TBinaryExpressionSyntax : TExpressionSyntax
     where TSimplifierOptions : SimplifierOptions
 {
-    private const string s_isPrefix = "Is";
-    private const string s_throwIfPrefix = "ThrowIf";
+    private const string IsPrefix = "Is";
```

Comment:
VS was constantly yelling at me for naming style violations. In this case I agreed (`s_` is for `static readonly` and pascal case is for `const`s)